src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1533,12 +1520,9 @@ MavlinkReceiver::handle_message_battery_status(mavlink_message_t *msg)
 	battery_status.connected = true;
 
 	// Get the battery level thresholds.
-	float bat_emergen_thr;
-	float bat_crit_thr;
-	float bat_low_thr;
-	param_get(_p_bat_emergen_thr, &bat_emergen_thr);
-	param_get(_p_bat_crit_thr, &bat_crit_thr);
-	param_get(_p_bat_low_thr, &bat_low_thr);
+	float bat_emergen_thr = _p_bat_emergen_thr.get();
```

Comment:
You could drop the local variables (bat_emergen_thr, bat_crit_thr, bat_low_thr) and use the params directly below.

Reply:
Done, along with a similar occurrence to eliminate `flow_rot_int`. Thanks!